scripts/services/properties.ts

```diff
@@ -0,0 +1,170 @@
+    return noPropertiesFound;
+
+  const [
+    controlTorAddress,
+    peerTorAddress,
+    macaroonHex,
+    macaroonBase64URL,
+    cert,
+  ] = await Promise.all([
+    ...paths.map(async (path) =>
+      (await effects.readFile({ volumeId: "main", path })).trim()
+    ),
+    effects.readFile({ volumeId: "main", path: "start9/admin.macaroon.hex" }),
+    effects.readFile({
+      volumeId: "main",
+      path: "start9/admin.macaroon.base64url",
+    }),
+    effects.readFile({
+      volumeId: "main",
+      path: "start9/control.cert.pem.base64url",
+    }),
+  ]);
+
+  try {
+    const nodeInfo = await effects.fetch(
+      "https://lnd.embassy:8080/v1/getinfo",
+      { headers: { "Grpc-Metadata-macaroon": macaroonHex } }
+    );
+    if (!nodeInfo.ok) return await compat.properties(effects);
+    const nodeInfoJson = await nodeInfo.json();
+    if (!nodeInfoMatcher.test(nodeInfoJson)) return wrongShape(nodeInfoJson);
+
+    const stats: T.Properties = {
+      version: 2,
+      data: {
+        "LND Sync Height": {
+          type: "string",
+          value: String(nodeInfoJson.block_height),
+          description: "The latest block height that has been processed by LND",
+          copyable: false,
+          qr: false,
+          masked: false,
+        },
+        "Synced To Chain": {
+          type: "string",
+          value: nodeInfoJson.synced_to_chain ? "✅" : "❌",
+          description:
+            "Until this value is ✅, you may not be able to see transactions sent to your on chain wallet.",
+          copyable: false,
+          qr: false,
+          masked: false,
+        },
+        "Synced To Graph": {
+          type: "string",
+          value: nodeInfoJson.synced_to_graph ? "✅" : "❌",
+          description:
+            "Until this value is ✅, you will experience problems sending payments over lightning.",
+          copyable: false,
+          qr: false,
+          masked: false,
+        },
+        "Node Alias": {
+          type: "string",
+          value: nodeInfoJson.alias,
+          description: "The friendly identifier for your node",
+          copyable: true,
+          qr: false,
+          masked: false,
+        },
+        "Node Id": {
+          type: "string",
+          value: nodeInfoJson.identity_pubkey,
+          description:
+            "The node identifier that other nodes can use to connect to this node",
+          copyable: true,
+          qr: false,
+          masked: false,
+        },
+        "Node URI": {
+          type: "string",
+          value: `${nodeInfoJson.identity_pubkey}@${peerTorAddress}:9735`,
+          description:
+            "Give this to others to allow them to add your LND node as a peer",
+          copyable: true,
+          qr: true,
+          masked: false,
+        },
+        "LND Connect gRPC URL": {
+          type: "string",
+          value: `lndconnect://${controlTorAddress}:10009?cert=${cert}&macaroon=${macaroonBase64URL}`,
```

Comment:
does this port need to change too?

Reply:
not sure. this port worked when I tested it as-is